### ISSUE TITLE
Framebuffer core depends on migen.actorlib

### ISSUE THREAD

**Issue by mithro** [m-labs/misoc#46]
As far as I can tell, migen.actorlib is no longer a thing?

https://github.com/m-labs/misoc/blob/master/misoc/cores/framebuffer/format.py#L6

**Comment by sbourdeauducq** [comment]
Yes. The framebuffer core and the dvi sampler haven't been tested for a long time so of course they're broken.
